CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## [Unreleased]
+
+### Fixed
+* Multiline parentheses objectExpression in indexer expression, 2176 [#2176](https://github.com/fsprojects/fantomas/issues/2176)
```

Comment:
The issue number is being duplicated here, this is inconsistent with all the other entries.

Reply:
Done, sorry.